npc/custom/item_options.txt

```diff
@@ -0,0 +1,234 @@
+//================= Hercules Script =======================================
```

Comment:
Good point about the sample folder, I always forget it exists.

My opinion is:

- If the script is meant to be an example for users that want to implement their own custom script, it should go into doc/sample.
- If the script is meant to be usable as is, it's fine to keep it in npc/custom - keeping in mind that it'll eventually be split to another repository and won't be maintained officially.